blocks/gmo-campaign-details/gmo-campaign-details.js

```diff
@@ -612,6 +601,126 @@ function switchTab(tab) {
     tab.classList.toggle('active');
 }
 
+function buildKPIList(program) {
+    let kpiList = document.createElement('ul');
+    program.primaryKpi?.forEach((kpi) => {
+        const kpiLi = createKPI(kpi);
+        kpiList.appendChild(kpiLi);
+    })
+    program.additionalKpi?.forEach((kpi) => {
+        const kpiLi = createKPI(kpi);
+        kpiList.appendChild(kpiLi);
+    })
+    if (kpiList.children.length == 0) {
+        kpiList.remove();
+        kpiList = document.createElement('div');
+        kpiList.textContent = "Not Available";
+    }
+    return kpiList;
+}
+
+function createKPI(kpi) {
+    const kpiLi = document.createElement('li');
+    const kpiText = parseString(kpi);
+    kpiLi.textContent = kpiText;
+    return kpiLi;
+}
+
+function buildProductList(program) {
+    const product = checkBlankString(program.productOffering);
+    const productList = document.createElement('div');
+    productList.classList.add('product', 'card-content');
+    /*
```

Comment:
remove this comment. Also, we may need to use this query to get the label
https://author-p108396-e1046543.adobeaemcloud.com/graphql/execute.json/gmo/getProductList

Reply:
Do we want to try that now, or have it as a to-do?